_includes/templates/corp-info-inst.html

```diff
@@ -0,0 +1,71 @@
+{%- comment -%}
```

Comment:
This include file is a clone. It's formal location should be in the "sample" folder of its owner component.

Check the example of gc-servinfo component and the temporary "sample" folder in "templates" folder